cmd/build/v1/build.go

```diff
@@ -91,9 +94,9 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 
 	buildMsg := fmt.Sprintf("Building '%s'", options.File)
 	if okteto.Context().Builder == "" {
-		oktetoLog.Information("%s using your local docker daemon", buildMsg)
+		bc.IoCtrl.Out().Infof("%s using your local docker daemon...", buildMsg)
```

Comment:
i'm confused, why it is not part of the logger? instead we have `bc.ioCtrl.Out()`

Reply:
Because we want the message in blue with the i squared. If we used the logger it will only appear if the log level is higher than the one we are printing.